app-modules/form/src/Models/SubmissibleAuthentication.php

```diff
@@ -66,7 +66,7 @@ public function prunable(): Builder
             ->where('created_at', '<', now()->subMonth());
     }
 
-    public function author(): MorphTo
+    public function author(): MorphTo|BelongsTo
```

Comment:
```suggestion
    public function author(): BelongsTo
```

MorphTo extends BelongsTo